core/controllers/voice_artist_test.py

```diff
@@ -416,3 +418,188 @@ def test_cannot_deassign_random_user_from_voice_artist(self):
         self.assertEqual(
             response['error'], 'Sorry, we could not find the specified user.')
         self.logout()
+
+
+class AudioUploaderHandlerTests(BaseVoiceArtistControllerTests):
+
+    EXP_ID = 'expId'
+
+    TEST_AUDIO_FILE_MP3 = 'cafe.mp3'
+    TEST_AUDIO_FILE_FLAC = 'cafe.flac'
+    TEST_AUDIO_FILE_OVER_MAX_LENGTH = 'cafe-over-five-minutes.mp3'
+    TEST_AUDIO_FILE_MPEG_CONTAINER = 'test-mpeg-container.mp3'
+    AUDIO_UPLOAD_URL_PREFIX = '/createhandler/audioupload'
+
+    def setUp(self):
+        super(AudioUploaderHandlerTests, self).setUp()
+        self.login(self.OWNER_EMAIL)
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
+        self.save_new_valid_exploration(
+            self.EXP_ID, self.owner_id, end_state_name='End card')
+        self.publish_exploration(self.owner_id, self.EXP_ID)
+        rights_manager.assign_role_for_exploration(
+            self.voiceover_admin,
+            self.EXP_ID,
+            self.voice_artist_id,
+            rights_domain.ROLE_VOICE_ARTIST
+        )
+
+        mock_accepted_audio_extensions = {
+            'mp3': ['audio/mp3'],
+            'flac': ['audio/flac']
+        }
+
+        self.accepted_audio_extensions_swap = self.swap(
+            feconf, 'ACCEPTED_AUDIO_EXTENSIONS',
+            mock_accepted_audio_extensions)
+
+        self.logout()
+
+        self.login(self.VOICE_ARTIST_EMAIL)
+
+        self.csrf_token = self.get_new_csrf_token()
+
+    def test_audio_upload_voice_artist(self):
```

Comment:
The test description is a bit confusing. If we are just testing a successful upload of audio file, can we have the test description reflect that?

Reply:
Here and below, please change the description so it follows the format `test_XXX_when_YYY`.